docs/CommerceEvent.md

```diff
@@ -1,22 +1,22 @@
-**session_id** | **Integer** |  | [optional] 
-**session_uuid** | **String** |  | [optional] 
-**custom_attributes** | **Hash&lt;String, String&gt;** |  | [optional] 
-**location** | [**GeoLocation**](GeoLocation.md) |  | [optional] 
-**device_current_state** | [**DeviceCurrentState**](DeviceCurrentState.md) |  | [optional] 
-**product_action** | [**ProductAction**](ProductAction.md) |  | [optional] 
-**promotion_action** | [**PromotionAction**](PromotionAction.md) |  | [optional] 
-**product_impressions** | [**Array&lt;ProductImpression&gt;**](ProductImpression.md) |  | [optional] 
-**shopping_cart** | [**ShoppingCart**](ShoppingCart.md) |  | [optional] 
-**currency_code** | **String** |  | [optional] 
-**screen_name** | **String** |  | [optional] 
-**is_non_interactive** | **BOOLEAN** |  | [optional] 
-
 
+| Name                      | Type                                                       | Description | Notes      |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ---------- |
+| **timestamp_unixtime_ms** | **Integer**                                                |             | [optional] |
+| **event_id**              | **Integer**                                                |             | [optional] |
+| **source_message_id**     | **String**                                                 |             | [optional] |
+| **session_id**            | **Integer**                                                |             | [optional] |
+| **session_uuid**          | **String**                                                 |             | [optional] |
+| **custom_attributes**     | **Hash&lt;String, String&gt;**                             |             | [optional] |
+| **location**              | [**GeoLocation**](GeoLocation.md)                          |             | [optional] |
+| **device_current_state**  | [**DeviceCurrentState**](DeviceCurrentState.md)            |             | [optional] |
+| **product_action**        | [**ProductAction**](ProductAction.md)                      |             | [optional] |
+| **promotion_action**      | [**PromotionAction**](PromotionAction.md)                  |             | [optional] |
+| **product_impressions**   | [**Array&lt;ProductImpression&gt;**](ProductImpression.md) |             | [optional] |
+| **shopping_cart**         | [**ShoppingCart**](ShoppingCart.md)                        |             | [optional] |
+| **currency_code**         | **String**                                                 |             | [optional] |
+| **screen_name**           | **String**                                                 |             | [optional] |
+| **is_non_interactive**    | **BOOLEAN**                                                |             | [optional] |
+| **custom_flags**          | **dict(str, str)**                                         |             | [optional] |
```

Comment:
Same feedback as above... seems likely that either `dict` here is wrong or Hash above should be changed to 'dict'